proc/tool/analysis/include/analysis.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * Parses csv files in a given directory and graphs data based
+ * on user specifications.
+ *
+ * @author Aaron Chan
+ */
+
+#ifndef DATA_ANALYSIS_ANALYSIS_H
+#define DATA_ANALYSIS_ANALYSIS_H
+
+#endif //DATA_ANALYSIS_ANALYSIS_H
```

Comment:
endif should go at the bottom of the file, after all you declarations. If i include this file twice I'll still get name collisions